matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/integrationmanager/AllowedWidgetsContent.kt

```diff
@@ -22,18 +22,20 @@ import com.squareup.moshi.JsonClass
 @JsonClass(generateAdapter = true)
 internal data class AllowedWidgetsContent(
         /**
-         * Map of stateEventId to Allowed
+         * Map of stateEventId to Allowed.
          */
         @Json(name = "widgets") val widgets: Map<String, Boolean> = emptyMap(),
 
         /**
-         * Map of native widgetType to a map of domain to Allowed
+         * Map of native widgetType to a map of domain to Allowed.
+         * <pre>
          * {
          *      "jitsi" : {
          *            "jitsi.domain.org"  : true,
          *            "jitsi.other.org"  : false
          *      }
          * }
+         * </pre>
```

Comment:
Missing dot here? Or it's not required when sentence starts and ends with same tag

Reply:
I think this is OK for detekt since the first sentence `Map of native widgetType to a map of domain to Allowed.` ends with a period.

Reply:
I just thought that every sentence in doc have to end with dot, since you actually do so in other places, with many sentences, like here:

 https://github.com/vector-im/element-android/blob/12eb23b198254191274601f0f933ceb67c7cb251/matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/query/QueryStringValue.kt#L50

It doesn't matter much, if it's ok for detekt, just curious